### DISCUSSION TITLE
Select multiple missing the (x) cancel

### DISCUSSION THREAD

**Discussion by bomshteyn** [filamentphp/filament#10529] [Help] [answered]
Here is my code:
```PHP
Select::make('genres')
    ->relationship('genres', 'name')
    ->searchable(['name'])
    ->multiple()
    ->preload()
    ->selectablePlaceholder(FALSE)
    ->createOptionForm([
        TextInput::make('name')
                 ->unique('genres', 'name')
                 ->required()
                 ->maxLength(255),
    ]),
```

The example on filament shows it with those small x icons here: https://filamentphp.com/docs/3.x/forms/fields/select#multi-select
<img width="584" alt="Screenshot 2023-12-27 at 2 36 54 PM" src="https://github.com/filamentphp/filament/blob/3.x/docs-assets/screenshots/images/light/forms/fields/select/multiple.jpg?raw=true">

But when i try using it i don't have the x for canceling the only way to remove is to backspace, which creates issue if i only want to remove one of the items in the middle: 

<img width="584" alt="Screenshot 2023-12-27 at 2 36 54 PM" src="https://github.com/filamentphp/filament/assets/4259699/d3b898a7-e61c-48f5-874b-e801243fd267">

**Comment by leandrocfe** [comment, marked as answer]
remove `->selectablePlaceholder`

**Reply by bomshteyn** [reply]
thanks
